### ISSUE TITLE
Attention mask values

### ISSUE THREAD

**Issue by hallogameboy** [allenai/longformer#22]
Hi,

In the tutorial of "how to use", the descriptions about the attention mask values are as follows:
```
# Attention mask values -- 0: no attention, 1: local attention, 2: global attention
```

But the comments about the attention mask in the code are:

https://github.com/allenai/longformer/blob/265314df581f8ec9e4cca98b27914583e8155905/longformer/longformer.py#L80-L91

The implementation seems to use the signs of attention mask values to control the masks, which is inconsistent with the tutorial.

Hence, I would like to ask for some clarifications and check the correct usage of attention masks.

Thanks very much, and congratulations on your impressive results! 

Best,
Jyun-Yu

**Comment by ibeltagy** [comment]
As the docstring says, the `attention_mask` is changed in `BertModel.forward` from 0, 1, 2 to -ve, 0, +ve. I will make the docstring more detailed because you are not the first one to mention this. Thanks.

**Comment by jyuno426** [comment]
@ibeltagy 
You meant that you edited the code of forward function in BertModel Class?
Then, I think you should notice how you edited it.

Thanks!

**Comment by ibeltagy** [comment]
@jyuno426, no, I am referring to [this function](https://github.com/huggingface/transformers/blob/master/src/transformers/modeling_bert.py#L636) which I am using with no changes.

**Comment by ibeltagy** [comment]
I will close this issue for now. Feel free to reopen if needed.
